src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -136,7 +135,7 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
 
     val world2 = (refundGasFn andThen payMinerForGasFn andThen deleteAccountsFn andThen persistStateFn)(resultWithErrorHandling.world)
 
-    TxResult(world2, gasUsed, resultWithErrorHandling.logs)
+    TxResult(world2, gasLimit - gasRefund, resultWithErrorHandling.logs)
```

Comment:
I see why I was confused: [this comment](https://github.com/input-output-hk/etc-client/blob/phase/3/txExecution/src/main/scala/io/iohk/ethereum/ledger/Ledger.scala#L346-L349) is inaccurate - it's the whole sum. Could you update it please?

Also the `gasRefund` name may be confusing - compare with `ProgramResult#gasRefund`. Maybe rename to `totalGasRefunded`. Also adding `val gasUsed = gasLimit - totalGasRefunded` could help readability, I think.